lib/modules/manager/npm/post-update/index.ts

```diff
@@ -377,7 +377,7 @@ export async function getAdditionalFiles(
   if (!packageFiles.npm?.length) {
     return { artifactErrors, updatedArtifacts };
   }
-  if (!config.updateLockFiles) {
```

Comment:
was that never strongly typed? 🤔

Reply:
Unfortunately not. `PostUpdateConfig` extends `Record<string, any>`.

https://github.com/renovatebot/renovate/blob/4047d64bc06f101956275db9c2750c6e07a4861f/lib/modules/manager/types.ts#L311-L312